File: Third semester/Synthesis and analysis of algorithms/Exercises/LinkedList/LinkedList/StartUp.cs
namespace LinkedList
{
    public class StartUp
    {
        public static void Main(string[] args)
        {
            LinkedList<int> myList = new LinkedList<int>();

            myList.InsertFront(5);
            myList.InsertFront(10);
            myList.InsertLast(15);

            Node<int> previousNode = new Node<int>(10);

            myList.InsertAfter(previousNode, 50);

            myList.PrintNodes();
        }
    }
}

File: Third semester/Java/Exercises/LinkedList/src/StartUp.java
public class StartUp {
    public static void main(String[] args)
    {
        LinkedList<Integer> list = new LinkedList<Integer>();
        list.insertFront(5);
        list.insertFront(6);
        list.insertFront(7);
        list.insertFront(8);
        list.insertFront(9);
        list.insertFront(10);

        System.out.println(list.size());
    }
}
